Description: Install something usable
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 quimup (1.3.1ppa1) precise quantal oneiric; urgency=low
 .
   * Tracking the upstream 1.3.1 release
   * Name changed from `guimup' to `quimup'
   * Modified XDG desktop file with proper categories and mime types
   * Checked using lintian and pbuilder for Debian Squeeze and
     Wheezy, Ubuntu Precise, Quantal, Raring and Saucy on i386 and
     amd64
Author: Ronald van Engelen <ronalde+launchpad@lacocina.nl>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- quimup-1.3.1ppa1.orig/quimup.pro
+++ quimup-1.3.1ppa1/quimup.pro
@@ -107,4 +107,8 @@ TRANSLATIONS += quimup_nl.ts \
 QT = core gui network
 
 XUP.SHOW_QT_VERSION_WARNING = 0
-XUP.OTHERS_PLATFORM_TARGET_DEFAULT = quimup
\ No newline at end of file
+XUP.OTHERS_PLATFORM_TARGET_DEFAULT = quimup
+
+TARGET = quimup
+target.path = /usr/bin
+INSTALLS += target
